program-test/src/lib.rs

```diff
@@ -223,7 +224,7 @@ impl solana_sdk::program_stubs::SyscallStubs for SyscallStubs {
 
     fn sol_invoke_signed(
         &self,
-        instruction: &Instruction,
+        instruction: &StableInstruction,
```

Comment:
this is breaking downstream-projects. This stubs stuff is public API so I think you'll have
to create the stable value internally